ent/schema/balance.go

```diff
@@ -0,0 +1,37 @@
+package schema
+
+import (
+	"entgo.io/contrib/entgql"
+	"entgo.io/ent"
+	"entgo.io/ent/schema"
+	"entgo.io/ent/schema/edge"
+	"entgo.io/ent/schema/field"
+)
+
+type Balance struct {
+	ent.Schema
+}
+
+// Fields returns Balance fields.
+func (Balance) Fields() []ent.Field {
+	return []ent.Field{
+		// account:contract
+		field.String("id").Unique().Immutable(),
+		field.Uint64("balance").Default(0),
```

Comment:
We should use a BigInt type here: https://github.com/dopedao/dope-monorepo/blob/master/packages/api/internal/ent/schema/wallet.go#L22